src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -15,22 +27,30 @@
     public static final String COMMAND_WORD = "find";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all patients whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "the specified keywords (case-insensitive),"
+            + "or whose phone number or nric matches and displays them as a list with index numbers.\n"
+            + "Parameters: [" + PREFIX_NAME + "NAME [MORE_NAMES]] "
```

Comment:
Would it sound better if `and displays them as a list with index numbers` is removed?  
What do you think?